cmd/ontap-cvo.go

```diff
@@ -171,7 +173,7 @@ func hashBucketName(name string) string {
 This will create the S3 bucket. Requires the bucketName to be hashed, the nasPath and relevant management and svm information
 https://docs.netapp.com/us-en/ontap-restapi/ontap/post-protocols-s3-buckets.html
 */
-func createS3Bucket(svmInfo SvmInfo, managementIP string, mgmInfo ManagementInfo, hashedbucketName string, nasPath string) error {
+func createS3Bucket(svmInfo SvmInfo, mgmUser string, mgmPassword string, managementIP string, hashedbucketName string, nasPath string) error {
```

Comment:
Nitpick: standardize `management` to `mgm`, unless there's a reason for the difference.